Cargo.toml

```diff
@@ -103,11 +103,11 @@ const_fn_assert = "0.1.2"
 nom = { version = "7.0.0", optional = true }
 new_debug_unreachable = "1.0.4"
 once_cell = "1.13.0"
-av1-grain = { version = "0.1.1", features = ["serialize"] }
+av1-grain = { version = "0.2.0", features = ["serialize"] }
```

Comment:
It seems that additional changes are required to bump av1-grain.

Reply:
Should be fixed now. I always forget all the myriad of commands that the CI runs so I just let it run them all for me. :sweat_smile: